trip-ninja-inc/trip_ninja_api_docs
# EPD-1888 | Flight doc response update

## LocalQA approved by


## Description
ClickUp Ticket: click [here](https://app.clickup.com/t/14197839/EPD-1888)

## Changes
* Updated fields for flight response where include_itineraries = true or false

## How to Test
1. Checkout this branch and run `npx @redocly/openapi-cli preview-docs tn_api_spec.yml`
2. Go to http://127.0.0.1:8080/ > Flight Request
3. Go through all the responses to make sure the fields match up to the postman responses.
